files/file.py

```diff
@@ -100,12 +100,14 @@
     group: foo
     state: link
 - file:
-    src: /tmp/{{ item.src }}
-    dest: "{{ item.dest }}"
+    src: '/tmp/{{ item.src }}'
+    dest: '{{ item.dest }}'
     state: link
   with_items:
-    - { src: 'x', dest: 'y' }
```

Comment:
I'm going to say that rewriting the `write_items` section here decreases readability, can you please revert that bit.